handsontable/src/3rdparty/walkontable/src/utils/nodesPool.js

```diff
@@ -27,9 +33,23 @@ export default class NodesPool {
   /**
    * Obtains an element. The returned elements in the feature can be cached.
    *
+   * @param {number} rowIndex The row index.
+   * @param {number} [columnIndex] The column index.
    * @returns {HTMLElement}
    */
-  obtain() {
-    return this.rootDocument.createElement(this.nodeType);
+  obtain(rowIndex, columnIndex) {
+    const hasColumnIndex = typeof columnIndex === 'number';
+    const key = hasColumnIndex ? `${rowIndex}x${columnIndex}` : rowIndex.toString();
+
+    if (this.pool.has(key)) {
+      return this.pool.get(key);
+    }
+
+    const node = this.rootDocument.createElement(this.nodeType);
+
+    // node.dataset.id = key; // Uncomment for debug purposes
```

Comment:
Not sure if this is meant to stay here for future use, or was just a temporary thing?

Reply:
done https://github.com/handsontable/handsontable/pull/11264/commits/c65364b9b787d789b33e8ebd800b7cdb0efbaa00